### PR TITLE
Making Terms of Service more visible

### PR DESCRIPTION
**Pull request by rabuono**
Added more explicit mentions of the Terms of Service section in the GDRP Compliance and Terms of Service page

### CODE REVIEW

**Review by bedroesb — APPROVED** [review]

File: gdpr-compliance-doc.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ All of the processing activities we currently do are required to use the service
 ### Right to Not be Subject to Automated Decision-making Including Profiling
 We do not profile our users, nor do any form of automated decision making based on personal data.
 
-## Terms of Service
+# Terms of Service
 
 ### Use of Service
```
Comment (bedroesb) [inline gdpr-compliance-doc.md:37]:
```suggestion
## Use of Service
```

File: gdpr-compliance-doc.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: GDPR Compliance Documentation
+title: GDPR Compliance and Terms of Service
 ---
 
 # GDPR Compliance Documentation
```
Comment (bedroesb) [inline gdpr-compliance-doc.md:5]:
```suggestion
# GDPR Compliance and Terms of Service
```

Comment (bedroesb) [reply gdpr-compliance-doc.md:5]: Title in the metadata is only for the html metadata but is not the Title that is displayed, this one is

Comment (rabuono) [reply gdpr-compliance-doc.md:5]: I wasn't clear. Sorry.
The idea was to have the two sections inside the same page. Hence not changing the heading level.
So a single `#` to make a GDPR and another for making a Terms of Service section.

File: gdpr-compliance-doc.md (hunk 1 above)
Comment (rabuono) [reply gdpr-compliance-doc.md:37]: Based on further discussion, resolving without commit

**Pull request merged**